CONTRIBUTOR-LADDER.md

```diff
@@ -0,0 +1,192 @@
+    responsibility, including Pull Requests that are assigned to them
+
+   * Helping other contributors become reviewers
+
+* Requirements:
+
+   * Is an Organization Member
+   * Has a track record of constructive and valuable reviews
+   * Has demonstrated knowledge of the specific area, for example by resolving
+   test failures or through contributions
+   * At least 10 Commits in the specific repository.
+   * Is supportive of new and occasional contributors and helps get useful PRs
+   in shape to commit
+
+* Privileges:
+
+   * Has GitHub rights to `lgtm` pull requests in specific directories
+   * Can recommend and review other Org Members to become Reviewers
+
+The process for an Organization Member to become a Reviewer is as follows:
+
+1. Open a PR in specific repository towards the `OWNERS` file and add the
+ persons github handle under `reviewers:` section.
+
+## Approver
+
+An Approver has similar responsibility for specific code, documentation,
+test, or other project areas asa Reviewer. An Approver is however expected to
+have deeper understanding of the code base of the specific repository or technical
+area than a Reviewer and also has shown a substantial track record of
+contribution and review in the project.
+
+Approvers are responsible for specific repositories and/or technical areas.
+Most often it is one or more related Git repositories. Approvers have all the
+rights and responsibilities of a Reviewer.
+
+* Responsibilities:
+
+   * Same as Reviewers
+   * Not only Review the PRs in the specific repositories but also engage/drive
+   in feature proposals and ensure release readiness of the repository.
+
+* Requirements:
+   * Same as Reviewers plus
+   * Has a track record of constructive and valuable reviews as a Reviewer
+   * Has demonstrated significant knowledge of the specific area, for example by
+   introducing new features in the repository and has substantial number of
+   commits in the repo or has a considerable amount of understanding of Metal3
+   ecosystem in case he/she is being considered as an Approver in a new
+   repository is being introduced in the project.
+   * Is supportive of new and occasional contributors and helps get useful PRs
+   in shape to commit
+
+* Privileges:
+   * Has GitHub rights to `approve` and `lgtm` pull requests in specific
+   directories
+   * Can recommend and review other Org Members to become Approvers
+
+The process for an Organization Member to become an Approver is as follows:
+
+1. One of the existing Approver can recommend adding a new Approver. The
+  recommendation is done by opening up an email discussion with the existing
+  Approvers.
+1. The email discussion should have an initial deadline within which the
+ existing Approvers can discuss on pros and cons on adding the specific person
+ as a Reviewer.
+1. Upon deadline based on the discussion outcome, if the new person can be
+  added as an Approver then open a PR in specific repository towards the
```

Comment:
This doesn't make clear who opens the PR. The only logical person is the approver recommending a new one.

Reply:
It should be better option if an approver opens the PR to add new approver or reviewer.